demisto_sdk/commands/validate/file_validator.py

```diff
@@ -503,11 +486,12 @@ def validate_added_files(self, added_files, file_type: str = None):  # noqa: C90
 
             elif checked_type(file_path, JSON_ALL_INCIDENT_TYPES_REGEXES):
                 incident_type_validator = IncidentTypeValidator(structure_validator)
-                if not incident_type_validator.is_valid_incident_type(validate_rn=not file_type):
+                if not incident_type_validator.is_valid_incident_type(validate_rn=False):
                     self._is_valid = False
 
             elif 'CHANGELOG' in file_path:
                 self.is_valid_release_notes(file_path)
+                continue
```

Comment:
why is that `continue` needed? 
if im not missing something, it would continue to the next iteration anyway